docs/source/data/data_catalog.md

```diff
@@ -667,6 +667,136 @@ You can use dataset factories to define a catch-all pattern which will overwrite
 Kedro will now treat all the datasets mentioned in your project's pipelines that do not appear as specific patterns or explicit entries in your catalog
 as `pandas.CSVDataSet`.
 
+### CLI commands for dataset factories
+
+To manage your dataset factories, two new commands have been added to the Kedro CLI: `kedro catalog rank` (0.18.12) and `kedro catalog resolve` (0.18.3).
+
+#### Using `kedro catalog rank`
+
+This command outputs a list of all dataset factories in the catalog, ranked by the priority that they are matched against. This priority is determined by considering the following criteria:
+
+1. The number of non-placeholder characters in the pattern
+2. The number of placeholers in the pattern
```

Comment:
```suggestion
2. The number of placeholders in the pattern
```

Reply:
It would be helpful to explain what placeholders are, in this case the curly brackets.